server/game/gamesteps/MeditatePrompt.js

```diff
@@ -173,6 +174,7 @@ class MeditatePrompt extends UiPrompt {
 
         if (!this.choosingPlayer.selectedDice.includes(die)) {
             this.choosingPlayer.setSelectedDice([die]);
+            this.prevDie = die.level; //When I tried =die, prevDie changed as the die level changed
```

Comment:
When card related events are created the source card is cloned and attached to the event / context
So you could do something similar here.
like this.prevDie = die.clone()
new clone method would just call new Die( with parameters from the existing die

Reply:
Thanks! I'll have a look into how to do this.